src/main/kotlin/racing/model/Car.kt

```diff
@@ -1,16 +1,28 @@
 package racing.model
 
-class Car {
-    private var moving: Int = 0
-    fun run(): Int {
-        val randomNumber: Int = (RANDOM_MIN_NUMBER..RANDOM_MAX_NUMBER).random()
-        if (randomNumber >= LIMIT_ADVANCE) moving++
-        return moving
+class Car(val name: String) {
+    companion object {
+        const val CAR_SYMBOL = "-"
+        private const val RANDOM_MIN_NUMBER = 0
+        private const val RANDOM_MAX_NUMBER = 9
+        private const val LIMIT_ADVANCE = 4
+        private const val MIN_NAME_LENGTH = 1
+        private const val MAX_NAME_LENGTH = 5
     }
 
-    companion object {
-        private const val RANDOM_MIN_NUMBER: Int = 0
-        private const val RANDOM_MAX_NUMBER: Int = 9
-        private const val LIMIT_ADVANCE: Int = 4
+    init {
+        require(name.length in MIN_NAME_LENGTH..MAX_NAME_LENGTH) {
+            "레이싱카 이름은 1글자 이상 5글자 이하입니다."
+        }
+    }
+
+    var position: Int = 0
```

Comment:
Car 객체의 상태 값을 외부에서 변경할 수 없도록 setter를 private 접근제한자로 제한하면 어떨까요? 😃
```suggestion
    var position: Int = 0
        private set
```